guides/common/modules/proc_applying-errata-to-a-host-collection.adoc

```diff
@@ -1,11 +1,29 @@
-[id="Applying_Errata_to_a_Host_Collection_{context}"]
+[id="applying-errata-to-a-host-collection"]
 = Applying errata to a host collection
 
-.Using `Remote Execution`
+You can use {Project} to apply errata to a host collection.
+
+ifndef::rest-api[]
```

Comment:
note: managing content guide shows with CLI and API procedures. Missing web and FAM procedures for now. related idea: https://github.com/theforeman/foreman-documentation/pull/3351#discussion_r1793641547